<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sim</class>
 <widget class="QDialog" name="Sim">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>771</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>771</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>771</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Link Simulator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>720</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_rate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Link Simulator Success Rate (%)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_rate">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>180</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Description</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_description">
      <property name="text">
       <string>The Link Simulator only affects outgoing PDUs. At any
given time, the Link Simulator can be in one particular
mode. The mode can be &quot;normal&quot;, &quot;duplicate&quot;,
&quot;reorder&quot;, or one of the &quot;drop&quot; modes. For more
information, see User Guide 3.1 Appendix B.</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_delay">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Delay between PDUs (microseconds)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_delay">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>210</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
   <property name="singleStep">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_modes">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>201</width>
     <height>451</height>
    </rect>
   </property>
   <property name="title">
    <string>Modes</string>
   </property>
   <widget class="QRadioButton" name="radioButton_duplicate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Duplicate</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_reorder">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Reorder</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropOneEOF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop One EOF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropOneFin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop One Fin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_normal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Normal</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAllACKAndFin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All ACK And Fin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAllFin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All Fin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAllNAK">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All NAK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAllButEOF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All But EOF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropAllButMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop All But MD</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropOneMD">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop One MD</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropOneACKEOF">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop One ACK EOF</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_dropOneACKFin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Drop One ACK Fin</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_corruptOptions">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>250</y>
     <width>201</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Corrupt Options</string>
   </property>
   <widget class="QRadioButton" name="radioButton_corruptNone">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Corrupt None</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_corruptEOFChecksum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Corrupt EOF
Checksum</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_corruptEOFFileSize">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Corrupt EOF
File Size</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>spinBox_rate</tabstop>
  <tabstop>spinBox_delay</tabstop>
  <tabstop>radioButton_normal</tabstop>
  <tabstop>radioButton_duplicate</tabstop>
  <tabstop>radioButton_reorder</tabstop>
  <tabstop>radioButton_dropAll</tabstop>
  <tabstop>radioButton_dropAllACKAndFin</tabstop>
  <tabstop>radioButton_dropAllFin</tabstop>
  <tabstop>radioButton_dropAllNAK</tabstop>
  <tabstop>radioButton_dropAllButEOF</tabstop>
  <tabstop>radioButton_dropAllButMD</tabstop>
  <tabstop>radioButton_dropOneACKEOF</tabstop>
  <tabstop>radioButton_dropOneACKFin</tabstop>
  <tabstop>radioButton_dropOneEOF</tabstop>
  <tabstop>radioButton_dropOneFin</tabstop>
  <tabstop>radioButton_dropOneMD</tabstop>
  <tabstop>radioButton_corruptNone</tabstop>
  <tabstop>radioButton_corruptEOFChecksum</tabstop>
  <tabstop>radioButton_corruptEOFFileSize</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Sim</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Sim</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
